### PR TITLE
System.Windows.Forms update

### PR DESCRIPTION
**Pull request by dreddy-work**
Fixes https://github.com/dotnet/winforms/issues/5167

@carlossanlop , I ran the tool as instructed in the repo wiki and these are the changes generated for dotnet/winforms repo.

Will check the preview link once checks passed to see everything still looks okay.

### CODE REVIEW

File: xml/System.Windows.Forms/MessageBoxButtons.xml [hunk 1]
```diff
@@ -1,206 +1,207 @@
...
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Windows.Forms.MessageBoxButtons</ReturnType>
+      </ReturnValue>
+      <MemberValue>6</MemberValue>
+      <Docs>
+        <summary>Specifies that the message box contains Cancel, Try Again, and Continue buttons.
+             This field is constant.</summary>
```
Comment (RussKie) [inline xml/System.Windows.Forms/MessageBoxButtons.xml:83]:
```suggestion
        <summary>Specifies that the message box contains Cancel, Try Again, and Continue buttons.</summary>

```

Link: https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.messageboxbuttons?view=net-6.0&branch=pr-en-us-6940
image

File: xml/System.Windows.Forms/Application.xml [hunk 2]
```diff
@@ -1,2205 +1,2225 @@
...
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">A main message loop is already running on this thread.</exception>
+        <altmember cref="M:System.Windows.Forms.Application.Exit" />
+        <altmember cref="M:System.Windows.Forms.Application.ExitThread" />
+        <altmember cref="M:System.Windows.Forms.Application.DoEvents" />
+      </Docs>
+    </Member>
+    <Member MemberName="Run">
+      <MemberSignature Language="C#" Value="public static void Run (System.Windows.Forms.ApplicationContext context);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig void Run(class System.
```
Comment (RussKie) [inline xml/System.Windows.Forms/Application.xml:1633]: This looks wonky
image

Link: https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.application.setdefaultfont?view=net-6.0&branch=pr-en-us-6940#remarks

File: xml/System.Windows.Forms/TaskDialogProgressBarState.xml [hunk 3]
```diff
@@ -1,154 +1,182 @@
...
+        </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+
+              Note: While the dialog is showing, you cannot switch from
+              <xref:System.Windows.Forms.TaskDialogProgressBarState.None> to any other state, and vice versa.
+
```
Comment (RussKie) [inline xml/System.Windows.Forms/TaskDialogProgressBarState.xml:131]: The preview doesn't have "Note:" 
image

Link: https://review.docs.microsoft.com/en-us/dotnet/api/system.windows.forms.taskdialogprogressbarstate?view=net-6.0&branch=pr-en-us-6940#fields

And even if it did, it'd look redundant
```suggestion
              While the dialog is showing, you cannot switch from
              <xref:System.Windows.Forms.TaskDialogProgressBarState.None> to any other state, and vice versa.
            

```

File: xml/System.Windows.Forms/AccessibleEvents.xml [hunk 4]
```diff
@@ -1,1064 +1,1084 @@
...
+          <para>A shortcut menu was displayed. The system sends this event for standard menus. Servers send this event for custom menus. The system does not send the event consistently.</para>
+          <para>For a call to <c>TrackPopupMenu()</c>, a client will see <see langword="EVENT_SYSTEM_MENUSTART" /> followed almost immediately by <see langword="EVENT_SYSTEM_MENUPOPUPSTART" /> for the popup being shown.</para>
+        </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Note that for a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART`
+             followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.
```
Comment (RussKie) [inline xml/System.Windows.Forms/AccessibleEvents.xml:813]:
```suggestion
             For a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART`
             followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.

```

image

File: xml/System.Windows.Forms/AccessibleEvents.xml [hunk 5]
```diff
@@ -1,1064 +1,1084 @@
...
+          <para>A shortcut menu was closed. The system sends this event for standard menus. Servers send this event for custom menus. When a shortcut menu is closed, the client receives this message followed almost immediately by the <see langword="SystemMenuEnd" /> event. The system does not send the event consistently.</para>
+          <para>For a call to <c>TrackPopupMenu()</c>, a client will see <see langword="EVENT_SYSTEM_MENUSTART" /> followed almost immediately by <see langword="EVENT_SYSTEM_MENUPOPUPSTART" /> for the popup being shown.</para>
+        </summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Note that for a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART`
+             followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.
```
Comment (RussKie) [inline xml/System.Windows.Forms/AccessibleEvents.xml:776]:
```suggestion
             For a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART`
             followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.

```

image

File: xml/System.Windows.Forms/MessageBoxButtons.xml (hunk 1 above)
Comment (dreddy-work) [reply xml/System.Windows.Forms/MessageBoxButtons.xml:83]: Tool exclude existing documented APIs but to make this reliable/consistent, I think we should do these changes in the source code in winforms repo and rerun this tool.

**Comment by RussKie** [comment]
@dreddy-work MC

**Comment by dreddy-work** [comment]
we don't need this any longer.

**Pull request closed without merging**